pkg/telemetry/telemetry.go

```diff
@@ -58,7 +58,7 @@ func CollectAnonymousCmdInfo() {
 	}
 }
 
-func machineID() string {
+func MachineID() string {
 	id, _ := generate()
```

Comment:
Could you check this one - it's generating id from machine name only so probably we'll have a lot of MacBook pro